includes/Integrations/Events_Calendar/Events_Calendar_Integration.php

```
@@ -0,0 +1,220 @@
+
+    /**
+     * Process TEC meta fields before saving
+     *
+     * @since WPUF_SINCE
+     *
+     * @param int   $post_id
+     * @param array $meta_key_value
+     * @param array $multi_repeated
+     * @param array $files
+     */
+    public function process_tec_meta_fields( $post_id, $meta_key_value, $multi_repeated, $files ) {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$multi_repeated'. (Unused Code Rules)

(UnusedFormalParameter)